### DISCUSSION TITLE
RMATS - get_rmats_coords

### DISCUSSION THREAD

**Discussion by onionpork** [dieterich-lab/Baltica#23] [Q&A]
Hi there,

I have a question that why only keep the introns that involves in the alternative splicing but excludes the canonical splicing introns?

**Comment by tbrittoborges** [comment]
Hi Yirui,

Just to clarify, are refering to [this function call](https://github.com/dieterich-lab/Baltica/blob/e2648cf0c305d46e731d0b93010d054c8d6aa431/baltica/parse_rmats_output.R#L100)?
